mathesar/models.py

```diff
@@ -309,6 +319,35 @@ def add_constraint(self, constraint_type, columns, name=None):
         return Constraint.objects.create(oid=constraint_oid, table=self)
 
 
+class Column(ReflectingObject):
+    table = models.ForeignKey('Table', on_delete=models.CASCADE, related_name='columns')
```

Comment:
I assume you'll add `attnum` and `display_options`?

Reply:
Yes, I pushed the changes I was sure of.